x/concentrated-liquidity/fees_test.go

```diff
@@ -603,22 +620,23 @@ func (s *KeeperTestSuite) TestCollectFees() {
 
 		// error cases.
 
-		"invalid pool id given": {
+		"accumulator does not exist": {
 			initialLiquidity: sdk.OneDec(),
 
 			lowerTickFeeGrowthOutside: sdk.NewDecCoins(sdk.NewDecCoin(ETH, sdk.NewInt(0))),
 			upperTickFeeGrowthOutside: sdk.NewDecCoins(sdk.NewDecCoin(ETH, sdk.NewInt(10))),
 
 			globalFeeGrowth: sdk.NewDecCoins(sdk.NewDecCoin(ETH, sdk.NewInt(10))),
 
-			owner:     ownerWithValidPosition,
-			lowerTick: 0,
-			upperTick: 1,
+			owner:       ownerWithValidPosition,
+			lowerTick:   0,
+			upperTick:   1,
+			frozenUntil: defaultFrozenUntil,
 
 			currentTick: 2,
 
 			isInvalidPoolIdGiven: true,
-			expectedError:        cltypes.PoolNotFoundError{PoolId: 2},
+			expectedError:        accum.AccumDoesNotExistError{AccumName: "fee/2"},
```

Comment:
Note to reviewer: `Require.ErrorAs()` was not picking this up correctly. I find this happens a lot with that and the only one that actually picks up Errors correctly is `ErrorContains`. Basically the error that was occurring was actually that the accumulator did not exist (since that pool was never created), not a `PoolNotFoundError`